src/app/pages/dashboard/dashboard.page.ts

```diff
@@ -0,0 +1,67 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { AlertController, Platform } from '@ionic/angular';
+import { Observable } from 'rxjs';
+
+import { DateTime } from '../../shared/helpers/date-time';
+import { DateTimeProvider } from '../../providers/date-time/date-time';
+import { LogoutBasePageComponent } from '../../shared/classes/logout-base-page';
+import { AuthenticationProvider } from '../../providers/authentication/authentication';
+import { StoreModel } from '../../shared/models/store.model';
+import { selectEmployeeName, selectVersionNumber } from '../../../store/app-info/app-info.selectors';
+import { LOGIN_PAGE } from '../page-names.constants';
+
+interface DashboardPageState {
+  appVersion$: Observable<string>;
+  employeeName$: Observable<string>;
+}
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: 'dashboard.page.html',
+  styleUrls: ['dashboard.page.scss'],
+})
+export class DashboardPage extends LogoutBasePageComponent {
+
+  pageState: DashboardPageState;
+  todaysDate: DateTime;
+  todaysDateFormatted: string;
+
+  constructor(
+    public alertController: AlertController,
```

Comment:
Would like us to use this migration opportunity to unify the visibility levels of the injected attributes. Everything should be `private` unless it has to be access in a derived class (`protected`) and there must be a very specific reason for a property to be `public`